packages/mobile/src/invite/saga.ts

```diff
@@ -192,6 +197,30 @@ export function* sendInvite(
       Logger.error(TAG, 'Currently only dollar escrow payments are allowed')
     }
 
+    // OPEN QUESTION - does storing invitee data only in the event of a successful transaction have
+    // unintended consequences? Seems right to me but previous position was before the generateStandbyTransactionId call
+
+    // OPEN QUESTION: EscrowedPayment and InviteDetails appear to store redundant data but invitee data is stored on send and escrow data is fetched from the smart contract.
+    // Should we consolidate into one data structure that we update when appropriate?
+    const inviteDetails: InviteDetails = {
+      timestamp: '1587621489078', // PLACEHOLDER - we probably want to read from the actual tx data for this
```

Comment:
Date.now() should be fine, it would be used in the UI that shows users their history